.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "python.testing.unittestArgs": [
+        "-v",
+        "-s",
+        "./python-sdk-sample",
+        "-p",
+        "test_*.py"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": true,
+    "python.testing.pytestArgs": [
```

Comment:
Related to `pytest`, please add it to the `requirements.txt` file since folks trying out the sample SDK will need to have it installed 🙂